server/.env

```diff
@@ -1,2 +1,4 @@
 TEAM_NAME="Jason Chui, Philippe Charpentier, Ziqi Shen, Milan Verma"
 FLASK_ENV="development"
+MONGODB_PWD="replace with DBAdminPWD provided on slack"
+TEST_ENV="Hello Team BLT"
```

Comment:
This file can be added to the .gitignore so it no longer shows up in the repository 👍